deploy/terraform/mod/k8s-operator/main.tf

```diff
@@ -0,0 +1,57 @@
+variable "image" {}
+
+resource "kubernetes_deployment" "deployment_msp_operator" {
+  metadata {
```

Comment:
Is it possible to load the these parameters from the yamls files we already have? Otherwise we'll know need to make sure we modify both everytime something changes.

Reply:
The whole idea is that you don't use YAML. Anything you want to change should be changed in `variables.tf`. And so you don't need to have YAML. Note; this is just for me, initially, and we don't have to merge it. But if there is some variance in configs, and you bump into something I do not. You send over the variables.tf I apply it using that and ... we should have the same, that's the idea. 

Plus, its destroy/apply with ordering.

Reply:
We don't need yaml if you install via terraform but for those who want to use yaml it would be nice to keep them in sync. Another possibility would be to generate yaml from this, rather than using helm for example. Is that possible at all?

Reply:
They don't seem to be interested in that, but here and there, there are some hacks and tools..

Reply:
Turns out there are external tools available to "from a running-config" generate YAML.

So you would apply config, and use a utility like kube-dump. You would lose, ofcourse, the ability to use variables and do "calculations".